src/main/resources/liquibase-changelog/versions/med-3.yaml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: 3
+      author: xxx
+      changes:
+        - createTable:
+            tableName: APP_USER
```

Comment:
Jak dodajesz tabele app_user to powinienes dodać również foreign key user_id dla Pacjenta oraz Doktora -> i usunac niepotrzebne juz w tym momencie kolumny, ktore znajduje sie w userze